src/lib/data/buyables/perdu.ts

```diff
@@ -1,293 +1,9 @@
-	{
-		name: "Explorer's ring 4",
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(["Explorer's ring 4"])
-	},
-	{
-		name: 'Morytania legs 1',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Morytania legs 1'])
-	},
-	{
-		name: 'Morytania legs 2',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Morytania legs 2'])
-	},
-	{
-		name: 'Morytania legs 3',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Morytania legs 3'])
-	},
-	{
-		name: 'Morytania legs 4',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Morytania legs 4'])
-	},
-	{
-		name: 'Varrock armour 1',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Varrock armour 1'])
-	},
-	{
-		name: 'Varrock armour 2',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Varrock armour 2'])
-	},
-	{
-		name: 'Varrock armour 3',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Varrock armour 3'])
-	},
-	{
-		name: 'Varrock armour 4',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Varrock armour 4'])
-	},
-	{
-		name: 'Wilderness sword 1',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Wilderness sword 1'])
-	},
-	{
-		name: 'Wilderness sword 2',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Wilderness sword 2'])
-	},
-	{
-		name: 'Wilderness sword 3',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Wilderness sword 3'])
-	},
-	{
-		name: 'Wilderness sword 4',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Wilderness sword 4'])
-	},
-	{
-		name: 'Western banner 1',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Western banner 1'])
-	},
-	{
-		name: 'Western banner 2',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Western banner 2'])
-	},
-	{
-		name: 'Western banner 3',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Western banner 3'])
-	},
-	{
-		name: 'Western banner 4',
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(['Western banner 4'])
-	},
-	{
-		name: "Rada's blessing 1",
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(["Rada's blessing 1"])
-	},
-	{
-		name: "Rada's blessing 2",
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(["Rada's blessing 2"])
-	},
-	{
-		name: "Rada's blessing 3",
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(["Rada's blessing 3"])
-	},
-	{
-		name: "Rada's blessing 4",
-		gpCost: 1000,
-		ironmanPrice: 200,
-		collectionLogReqs: resolveItems(["Rada's blessing 4"])
-	}
-];
+export const perduBuyables: Buyable[] = diariesCL.map(itemName => ({
+	name: itemName,
+	gpCost: 1000,
+	ironmanPrice: 200,
+	collectionLogReqs: [itemName]
```

Comment:
you cant change this to be just strings/item names. (1) any incorrect item names wont cause any error until someone actually tries to buy one (2) its a standard that we always use item IDs, using `resolveItems()`, for things like this.

Reply:
It worked in testing so I assumed it was good, my apologies. I think I have made the relevant changes, and it still works as it should in testing